Cargo.toml

```diff
@@ -20,7 +20,7 @@ maintenance = { status = "experimental" }
 travis-ci = { repository = "https://github.com/nixpulvis/oursh" }
 
 [features]
-default = ["raw", "shebang-block"]
+default = ["raw", "history", "completion", "shebang-block"]
 
 # TODO: Justify and explain features.
```

Comment:
As always, I think documentation will mark the end of the PR.

Generally in a collaborative environment (which this is currently not), I like this, writing a summary gives people time to review as well as giving future developers the needed information.